patches/focus-trap+7.5.4.patch

```diff
@@ -0,0 +1,88 @@
+diff --git a/node_modules/focus-trap/dist/focus-trap.esm.js b/node_modules/focus-trap/dist/focus-trap.esm.js
+index 10d56db..a0ed057 100644
+--- a/node_modules/focus-trap/dist/focus-trap.esm.js
++++ b/node_modules/focus-trap/dist/focus-trap.esm.js
+@@ -100,8 +100,8 @@ var isKeyForward = function isKeyForward(e) {
+ var isKeyBackward = function isKeyBackward(e) {
+   return isTabEvent(e) && e.shiftKey;
+ };
+-var delay = function delay(fn) {
+-  return setTimeout(fn, 0);
++var delay = function delay(fn, delayTime = 0) {
++  return setTimeout(fn, delayTime);
```

Comment:
I think we need to keep the extra setTimeout here and add our timeout. After testing I found out that document.activeElement does not return the correct focused element.

```suggestion
+  return setTimeout(() => setTimeout(fn, delayTime), 0);
```

Reply:
Can you please elaborate what does this has to do with focus deactivation? What I am referring to is the fact that if you log `document.activeElement` you will not get the correct focused element as we are doing this too early


https://github.com/Expensify/App/assets/16493223/bc39cb75-1f9e-4ed1-9d78-302d3ffb05a1

Reply:
The correct timing is 300ms (`CONST.ANIMATED_TRANSITION`). `CONST.ANIMATION_IN_TIMING` is not related to the screen transition.

Reply:
Given that this already had it's own `setTimeout` even when no timeout is actually required, we should just add another `setTimeout`. The former is to fix whatever the library was trying to fix and the later is for our actual wanted delay.